src/runtime_src/tools/xclbinutil/aie-pdi-transform/.clang-tidy

```diff
@@ -0,0 +1,295 @@
+---
```

Comment:
Do we need this separate copy of .clang-tidy? I believe clang-tidy recursively searches for .clang-tidy in the parent directories so it should find the one in the root of the project.

Reply:
This .clang-tidy is different from the one in the root directory. I added a comment a while ago.
> I used the .clang-tidy at the root directory as the template and removed the checkers related to C++ (as shown below)
> cppcoreguidelines-*, hicpp-*, -hicpp-no-array-decay, -hicpp-signed-bitwise, -cppcoreguidelines-pro-bounds-pointer-arithmetic, -cppcoreguidelines-pro-type-union-access, -cppcoreguidelines-pro-bounds-array-to-pointer-decay, -cppcoreguidelines-pro-type-reinterpret-cast, -cppcoreguidelines-pro-bounds-constant-array-index,